### PR TITLE
Created objectives.md

### PR DESCRIPTION
**Pull request by FarhanMajumder**
I made the objectives.md file in the /doc folder.

### CODE REVIEW

**Review by RuiZhou96 — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

File: doc/objectives.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "objectives.md"
```
Comment (lwjohnst86) [inline doc/objectives.md]: This should be "Objectives"

**Comment by lwjohnst86** [comment]
Great! @FarhanMajumder if you just fix the title, you can merge it in.

**Review by RuiZhou96 — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

File: doc/objectives.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Objectives.md"
```
Comment (QuLogic) [inline doc/objectives.md]: No ".md" either. This is the title of the document.

**Review by leilakrichel — APPROVED** [review]

File: doc/objectives.md (hunk 2 above)
Comment (FarhanMajumder) [reply doc/objectives.md]: Ok understood.

**Review by RuiZhou96 — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

**Pull request merged**